pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pydantic-avro"
-version = "0.3.2"
+version = "0.4.0"
```

Comment:
can you please revert this version change?
Will do this is a release

Reply:
sure